lib/AST/LocalizationFormat.cpp

```diff
@@ -101,6 +101,7 @@ readYAML(llvm::yaml::IO &io, T &Seq, bool, Context &Ctx) {
       DiagnosticNode current;
       yamlize(io, current, true, Ctx);
       io.postflightElement(SaveInfo);
+      std::replace(current.msg.begin(), current.msg.end(), '\n', ' ');
```

Comment:
@xedin The parsed multiline YAML string will have line breaks `\n` in it. So, I had to manually replace them with spaces.

Reply:
Should this also check for other kinds of line breaks, such as `\r\n` (on Windows)?

Reply:
I don't have a Windows machine so I don't really know. How should I test it?

Reply:
Let's not do this at all for now, we can validate YAML format to detect such problems early.